src/View/pagination.html.twig

```diff
@@ -0,0 +1,58 @@
+<html>
+<head>
+  <title>Message Board</title>
+</head>
+<body>
```

Comment:
if this file is designed to be used as an include, you don't need these above tags (or their closing tags at the bottom of the file). Twig literally takes everything in this file and dumps it where you include it so when you include this in another webpage, these tags will also get included somewhere they are invalid.